REST-API.md

```diff
@@ -335,8 +336,197 @@
+        <th>Body</th>
+        <td>void</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>void</td>
+    </tr>
+</table>
+
+### Index ready - I don't know what it does
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/index_ready</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>GET</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td>none</td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>void</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>boolean</td>
+    </tr>
+</table>
+
+### Enable/Disable indexation
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/indexation</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>PUT</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td><ul>
+            <li>enabled</li>
+        </ul></td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>void</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>void</td>
+    </tr>
+</table>
+
+### Enable/Disable instrumentation
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/instrumentation</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>PUT</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td><ul>
+            <li>enabled</li>
+        </ul></td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>void</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>void</td>
+    </tr>
+</table>
+
+### Check if instrumentation is enabled
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/instrumentation</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>GET</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td>none</td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>void</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>boolean</td>
+    </tr>
+</table>
+
+### Dependent Zones - I don't know what it does
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/dependent_zones</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>POST</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td>none</td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>JSON: list(string)</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>JSON: {"zones": list(string)}</td>
+    </tr>
+</table>
+
+---
+
+## Load simulation
+
+### Run batch of operations
+
+<table>
+    <tr>
+        <th>Path</th>
+        <td>/simulate_load</td>
+    </tr>
+    <tr>
+        <th>Method</th>
+        <td>POST</td>
+    </tr>
+    <tr>
+        <th>Params</th>
+        <td>none</td>
+    </tr>
+    <tr>
+        <th>Body</th>
+        <td>JSON {"ops": [{}, ..]}<br/>Where inner objects have fields &lt;"t" - operation type {"a", "r", "p"}, "f" - from, "to" - to, "p" - permissions, "tr" - trace&gt;</td>
+    </tr>
+    <tr>
+        <th>Return type</th>
+        <td>void</td>
+    </tr>
+</table>
+
+---
+
 ## Empty table template
```

Comment:
To chyba nie jest konieczne

Reply:
Co nie jest konieczne? Empty table template? Mi się to przydaje bo jak dopisuję to po prostu ją kopiuje i mam to puste, nie muszę kasować z jakiegoś innego. Dopóki nie skończę tego pliku, to to zostaje